pages.zh/linux/grub-install.md

```diff
@@ -0,0 +1,16 @@
+# grub-install
+
+> 安装 GRUB 到设备。
+> 更多信息：<https://www.gnu.org/software/grub/manual/grub/html_node/Installing-GRUB-using-grub_002dinstall.html>。
```

Comment:
```suggestion
> 更多信息：<https://www.gnu.org/software/grub/manual/grub/html_node/Installing-GRUB-using-grub_002dinstall.html>.
```
After Latin text, we use Latin punctuation

Reply:
understand!